onnxruntime/core/common/threadpool.cc

```diff
@@ -290,7 +290,6 @@ static ptrdiff_t CalculateParallelForBlock(const ptrdiff_t n, const Eigen::Tenso
     if (coarser_efficiency + 0.01 >= max_efficiency) {
       // Taking it.
       block_size = coarser_block_size;
-      block_count = coarser_block_count;
```

Comment:
> block_count = c [](start = 5, length = 16)

Is this necessary?

Reply:
we are removing the line since Xcode complains that "block_count" not consumed anywhere  in downstream.